lib/v2/screens/transfer/send/send_confirmation/interactor/mappers/send_transaction_state_mapper.dart

```diff
@@ -16,48 +14,28 @@ class SendTransactionStateMapper extends StateMapper {
       return currentState.copyWith(pageState: PageState.failure, errorMessage: result.asError!.error.toString());
     } else {
       final resultResponse = result.asValue!.value as SendTransactionResponse;
-      final transactionId = resultResponse.transactionId.asValue!.value as String;
 
```

Comment:
this has the most changes since it now handles 2 different cases: 

1. Transfer (same as before)
2. Generic transaction (new)